code/__defines/subsystems.dm

```diff
@@ -43,6 +47,15 @@
 	}\
 }
 
+/**
+	Create a new timer and add it to the queue.
+	* Arguments:
+	* * callback the callback to call on timer finish
+	* * wait deciseconds to run the timer for
+	* * flags flags for this timer, see: code\__DEFINES\subsystems.dm
+*/
+#define addtimer(args...) _addtimer(args, file = __FILE__, line = __LINE__)
```

Comment:
this macro already existed, has it been moved or remade or duplicated?
does this version add anything meaningful?

Reply:
> Turns /proc/addtimer into /proc/_addtimer and creates an addtimer macro in its place. This is because the macro will pass in the file and line of what is creating the timer. I wanted to use gib_stack_trace() and friends, but those regularly turned up blank exception descriptions, and were useless. If there's anything better, let me know, but for now this is better than nothing.
>
> The command will only show those still in the buckets. This is to avoid both leaving too much of a footprint on the addtimer proc and due to memory concerns of logging all timers being created.

– PR where it was added. 
It's used to track where timer was added.